### ISSUE TITLE
How to increase GPU occupancy? Reasoning can only use 2GB?

### ISSUE THREAD

**Issue by ss-lb** [neuralchen/SimSwap#283]
![image](https://user-images.githubusercontent.com/107528966/173984640-e1a578c1-0070-47b5-961c-0d4a82e7707c.png)

**Comment by skintflickz** [comment]
yes would like to know how to increase GPU usage to speed up processing?

**Comment by monument-and-sea-all-the-gift** [comment]
> yes would like to know how to increase GPU usage to speed up processing?

make bachsize bigger

**Comment by Akyariss** [comment]
> make bachsize bigger

How can you increase batch size?

**Comment by ichason** [comment]
yes would like to know how to increase GPU usage to speed up processing?

**Comment by monument-and-sea-all-the-gift** [comment]
> yes would like to know how to increase GPU usage to speed up processing?

train or inference
